dev/com.ibm.ws.microprofile.config.1.1_fat/fat/src/com/ibm/ws/microprofile/config/fat/tests/BasicConfigTests.java

```diff
@@ -76,9 +77,14 @@ public class BasicConfigTests extends FATServletClient {
     public static final String CUSTOM_SOURCES_APP_NAME = "customSources";
     public static final String TYPES_APP_NAME = "types";
 
-    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP50, MicroProfileActions.MP12, MicroProfileActions.MP13, MicroProfileActions.MP14,
+    @ClassRule
+    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP60,
```

Comment:
MicroProfileActions.repeat will need an update to work on Java 8 because it marks the first FeatureSet for lite and the rest for full.  But with Java 8 then no tests will run and the build will break.

Otherwise you do what we have done in other FATs if Java 8 use MP 5 as first feature set and don't have mp5 in the other feature sets lists.  If I am not making sense, let me know.

Reply:
Ah, good point, thank you

Reply:
Updated, I've added a new `MicroProfileActions.repeat` method which substitutes the most recent compatible set from `otherFeatureSets`.